usagov/vote-gov-drupal
# VOTE-974: Enable contrib module Node Revision Delete

## Jira ticket
[VOTE-974](https://bixal-projects.atlassian.net/browse/VOTE-974)

## Description
Enable contrib module Node Revision Delete

## Deployment and testing (required, if applicable)
### Pre-deploy steps
N/A

### Post-deploy steps
N/A

### Testing steps
Go to any Content Type, select Publishing Options tab, check Limit Node Revisions.
